stac_nb/stac_nb.py

```diff
@@ -115,7 +139,7 @@ def click_button(self, b):
 
         query_response = self.client.search(**payload_dict)
 
-        self.query_results = list(query_response.get_items())
+        self.query_results = VisualList(query_response.get_items())
```

Comment:
Could be that this line needs to be changed to:
```suggestion
        self.query_results = VisualList(list(query_response.get_items()))
```
I'm actually not as experienced in Python as it might look like. ;-)